CONTRIBUTING.md

```diff
@@ -46,14 +46,8 @@ Thus for development it is necessary to checkout a copy of such a dependency and
 
 When running an example using `yarn start-local` you can use local version of [luma.gl](https://github.com/visgl/luma.gl/) or [math.gl](https://github.com/uber-web/math.gl) by appending the `--env.local-luma` or `--env.local-math` option.
 
-### Specific module overrides
+It is possible to test against the local source of other dependency libraries by modifying [examples/vite.config.local.mjs](https://github.com/visgl/deck.gl/blob/master/examples/vite.config.local.mjs).
```

Comment:
Do you know if local webpack setups, like [kepler.gl's webpack.config.local.js](https://github.com/keplergl/kepler.gl/blob/master/examples/webpack.config.local.js), could still work after our transition to `vite`?

Reply:
I don't see a reason why not...